### PR TITLE
Care location enahancement

### PR DESCRIPTION
**Pull request by kushal-arora-fw**
This PR includes the following changes :

1. **User can now select CareLocation specific care activities**
            * [Care Locations / Unit] [DB] updates: New relation with CareAcvity with migration with definition and data updates. Seed service changes
            * [Care Locations / Unit] [DB] [API] updates: New relation with planning session; Migration and API updated
            * [CareLocation] [API] update api changes to support care activities selection based on chosen care location; + Seed changes
            * [CareLocation] [WEB] update FE changes to support care activities selection based on chosen care location

2. **Removing Occupation Tag**

### CODE REVIEW

**Review by dbayly-freshworks — APPROVED** [review]
A few nits but overall looks good. I appreciate the comments you added.

File: apps/web/src/components/OccupationItem.tsx [hunk 1]
```diff
@@ -10,17 +10,22 @@ export const OccupationItem = ({ id, displayName, isRegulated }: OccupationItemP
...
-      <div className='flex flex-initial w-1/6 justify-end'>
+
+      {/** No need to show occupation tag: All occupations are regulated for now. */}
+      {/** changes: (1/2) w-5/6 changed to w-full for Checkbox's parent div */}
+      {/** changes: (2/2) commented Tag element */}
+
+      {/* <div className='flex flex-initial w-1/6 justify-end'>
         <Tag text={tagText} tagStyle={tagColor}></Tag>
-      </div>
+      </div> */}
```
[the remark this answers is not in the document]
Comment (kushal-arora-fw) [reply apps/web/src/components/OccupationItem.tsx:28]: Right, but I anticipate this code coming back.
We will, for now, won't show "Unregulated" occupations; and hence this code doesn't make sense. But perhaps in future, we will. It's all work done, and I do not want future dev to spend time on same task. 
What do you think?

File: apps/web/src/services/useCareActivities.ts [hunk 2]
```diff
@@ -2,22 +2,26 @@ import { useEffect, useState } from 'react';
 import { API_ENDPOINT } from '../common';
 // import { OptionType } from '../components/generic/RenderSelect';
```
[the remark this answers is not in the document]
Comment (kushal-arora-fw) [reply apps/web/src/services/useCareActivities.ts]: sure!

File: apps/api/src/care-activity/care-activity.service.ts [hunk 3]
```diff
@@ -13,6 +15,35 @@ export class CareActivityService {
     private readonly careActivityRepo: Repository<CareActivity>,
   ) {}
 
+  async getCareActivitiesByBundlesForCareLocation(careLocationId: string): Promise<BundleRO[]> {
+    if (!careLocationId) {
+      throw new NotFoundException({ message: 'Care Location Not found' });
```
[the remark this answers is not in the document]
Comment (kushal-arora-fw) [reply apps/api/src/care-activity/care-activity.service.ts:20]: User wouldn't be reaching this point without providing the CareLocationID; Data input errors is something DTOs will handle.

This use case is when the user has selected a careLocation in a previous step, and somehow DB isn't able to find it [perhaps never].

**Pull request merged**